src/main/java/com/woowacamp/soolsool/core/member/controller/MemberController.java

```diff
@@ -64,6 +65,17 @@ public ResponseEntity<ApiResponse<Void>> removeMember(
         memberService.removeMember(userId);
 
         return ResponseEntity.status(HttpStatus.NO_CONTENT)
-            .body(ApiResponse.of(MemberResultCode.MEMBER_DELETE_SUCCESS, null));
+            .body(ApiResponse.from(MemberResultCode.MEMBER_DELETE_SUCCESS));
+    }
+
+    @PatchMapping("/mileage")
+    public ResponseEntity<ApiResponse<Void>> addMemberMileage(
```

Comment:
```suggestion
    public ResponseEntity<ApiResponse<Void>> modifyMemberMileage(
```

마일리지를 수정만 하는 로직이니 modify가 적절해 보여요.

Reply:
아니면 charge 는 어때?? modify라는게 mileage 차감도 포함하는 포괄적인 의미인것 같아서 우선 add로 하긴 했는데 add 딱히 의미가 명확해 보이진 않고 차라리 충전한다는 의미인 charge도 괜찮아 보이는데?

Reply:
그럼 마일리지를 사용에 대한 endpoint는 `PACTH /mileage`랑 다른걸 사용하나요?

Reply:
마일리지 사용은 endpoint가 따로 없고 order에 대한 요청이 이뤄질때 business logic 에서 결제가 완료되면 차감 하는 방식으로 생각했었어~

Reply:
아.. 소까.. 그럼 charge도 죠씁니다